packages/breadboard-ui/src/elements/input/input.ts

```diff
@@ -49,7 +54,12 @@ const parseValue = (type: Schema["type"], input: HTMLInputElement) => {
 };
 
 @customElement("bb-input")
-export class Input extends LitElement {
+export class Input extends LitElement implements BreadboardWebElement {
```

Comment:
nit: I don't think we need to implement an interface here; it's an HTMLElement which should dispatch Events, and its parent element should catch and handle appropriately. I don't think we gain much by insisting on a particular interface.